app/src/androidTest/java/com/github/mateo762/myapplication/HabitsActivityFragmentNavigationTest.kt

```diff
@@ -30,6 +30,7 @@ class HabitsActivityFragmentNavigationTest {
         activityScenario.close()
     }
 
+/*
```

Comment:
Maybe we can mark this and delete it in a future commit to avoid having a lot of dead code in the project?

Reply:
Yes, the week calendar is very tricky to test, and for some reason the test gets stuck. I left it in case in the future there is time to fix things, we can try to recover the test but right now I don't think its worthy spending so much time on it.